ROADMAP.md

```diff
@@ -1,112 +1,101 @@
-# Roadmap: August - October 2020
+# Roadmap:  November 2020 - January 2021
 
 Every few months, the team will highlight areas of focus for our work and upcoming research.
 
-We select items for inclusion in the roadmap from the Top 10 Community Issues, [core services](docs/CORE_SERVICES.md), and internal priorities. When community pull requests exist for a given item, we will prioritize working with the original authors to include their contributions. If the author can no longer take on the implementation, HashiCorp will complete any additional work needed.
+We select items for inclusion in the roadmap from the Top 10 Community Issues, [core services](docs/CORE_SERVICES.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist, or the original contributors are not available we will create the resources and implementation ourselves.
```

Comment:
Change `core services` to `Core Services` to match capitalization later in the document.